package.json

```diff
@@ -4,11 +4,12 @@
   "license": "MIT",
   "description": "",
   "scripts": {
-    "test": "jest -c test/jest.config.js",
+    "test": "jest",
```

Comment:
No need to specify custom config file since it's now stored in the default path (`<root_dir>/jest.config.js`)